lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -243,6 +249,32 @@ def get_progress(self, model: CourseEnrollment) -> Dict[str, int]:
             'num_points_possible': sum(map(lambda x: x.graded_total.possible if x.graded else 0, subsection_grades)),
         }
 
+    def get_course_assignments(self, model: CourseEnrollment) -> Optional[Dict[str, List[Dict[str, str]]]]:
+        """
+        Returns the future assignment data and past assignments data for the user in the course.
+        """
+        assignments = get_course_assignment_date_blocks(
+            self.course,
+            model.user,
+            self.context.get('request'),
+            include_past_dates=True
+        )
+        next_assignment = {}
```

Comment:
```suggestion
        next_assignment = None
```
Maybe we will declare it with `None` and return also return `None` if no future assignment?

Reply:
Fixed